### DISCUSSION TITLE
Where can i find html templates for this tool?

### DISCUSSION THREAD

**Discussion by xMidNightFTW** [KnugiHK/WhatsApp-Chat-Exporter#87] [Q&A]
Is there a whatsapp look like template/theme that is compatible with this tool?

I tried making it with css and html but it seems w3 css library is being used which makes it kinda tricky.. So i was wondering if there's anything ready.

**Comment by KnugiHK** [comment]
Perhaps the only template available is the one included.

You can always create a template from scratch. But you will need to study the code a bit, especially the classes in [data_model.py](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/blob/main/Whatsapp_Chat_Exporter/data_model.py). The [customization documentation](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/wiki/Result-HTML-Customization-(Docs-In-Progress)) is still in progress, but you might still find it handful.

I planned to move to Bootstrap, but there is no timeline for that.

**Reply by xMidNightFTW** [reply]
Alright I will take a look

Thank you for the tool and for the quick response!

**Comment by KnugiHK** [comment]
Continue on #97.
